UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -754,18 +761,33 @@ public void RestartRound()
 		StartCoroutine(ServerRoundRestart());
 	}
 
-	IEnumerator ServerRoundRestart()
+	private float GetMemeoryUsagePrecentage()
 	{
-		Logger.Log("Server restarting round now.", Category.Round);
-		Chat.AddGameWideSystemMsgToChat("<b>The round is now restarting...</b>");
+		return (Profiler.GetTotalAllocatedMemoryLong() / 1048576) / SystemInfo.systemMemorySize * 100;
```

Comment:
1048576)?

Reply:
GetTotalAllocatedMemoryLong() returns memory in bytes; this converts it into MBs

Reply:
ok cool